src/server/identification/ear-tags-calves/index.js

```diff
@@ -0,0 +1,39 @@
+import { QuestionPageController } from '../../common/controller/question-page-controller/question-page-controller.js'
+import { EarTagsAnswer } from '../../common/model/answer/ear-tags/ear-tags.js'
+import { QuestionPage } from '../../common/model/page/question-page-model.js'
+import { cattleOver42DaysPage } from '../cattle-over-42-days-old/index.js'
+
+const customHeading = 'Official ear tag numbers for calves under 42 days old'
+
+export class EarTagsCalvesPage extends QuestionPage {
+  urlPath = '/identification/enter-ear-tags-calves'
+  sectionKey = 'identification'
+  question =
+    'Enter the ear tag numbers of the calves under 42 days old you are planning to move'
+
+  questionKey = 'earTagsCalves'
+  Answer = EarTagsAnswer
```

Comment:
I think the errors are wrong because you are reusing EarTagsAnswer here and I guess it has to be a different model with its own error messages